Sources/XiEditor/EditViewController.swift

```diff
@@ -711,6 +717,10 @@ class EditViewController: NSViewController, EditViewDataSource, FindDelegate, Sc
         document.xiCore.setTheme(themeName: sender.title)
     }
     
+    @IBAction func debugToggleTail(_ sender: NSMenuItem) {
+        document.xiCore.toggleTailConfig(identifier: document.coreViewIdentifier!, enabled: !self.isTailEnabled)
```

Comment:
We could probably rewrite this in a way so we don't have to do `!self.isTailEnabled` here, since it feels counter intuitive.

Reply:
Rewrite how?